contracts/Comptroller.sol

```diff
@@ -365,6 +368,14 @@ contract Comptroller is ComptrollerV3Storage, ComptrollerInterface, ComptrollerE
             return uint(Error.PRICE_ERROR);
         }
 
+        // Error if borrow brings total borrow above limit
+        (MathError mathErr, uint hypotheticalTotalBorrows) = addUInt(CToken(cToken).totalBorrows(),borrowAmount);
```

Comment:
I believe it'd be better to use `0` as the "no limit case", in which case this could basically be:
```solidity
uint borrowLimit = borrowLimits[cToken];
if (borrowLimit > 0) {
    uint totalBorrows = CToken(cToken).totalBorrows();
    (MathError mathErr, uint nextTotalBorrows) = addUInt(totalBorrows ,borrowAmount);
    require(mathErr == MathErorr.NO_ERROR, "Borrow limit overflow");
    
    if (nextTotalBorrows > borrowLimit) {
        return uint(Error.MARKET_BORROW_LIMIT_REACHED);
    }
}
```

Reply:
Yes now that I think of it, I agree. Initially, I was going to have default be 0 with a limit of 0 borrows but later decided I didn't want to have to dig through all the tests so changed it to no limit. Makes much more sense having 0 be unlimited now.